### DISCUSSION TITLE
Thank you to everyone contributing.

### DISCUSSION THREAD

**Discussion by Jwn5k** [OpenLoco/OpenLoco#1678] [General]
I want to say thank you to all of those contributing to the development of OpenLoco. I have loved fiddling around in Locomotion for many years, and I value all the time spend developing this project. I am hopeful MP will be fixed some day, but I understnad development must focus elsewhere for now. I highly appreciate all the work that has been done so far, love the communities around games like this, RCT, and TTD. Thank you.

**Comment by hikayamasan353** [comment]
Anyone remembers this: "Error: OpenLoco 0.1. does not exist. If you have run this on 1st of April, you'd have known that :D"? I'm still emotionally traumatized by this prank. I know that development isn't easy and takes alot of time, but this prank from the OpenTTD forum will still haunt me for life that even this project won't relieve anything.
Still it's awesome that people are actually working on this. I own a copy of the original Locomotion and still listen to music from it such as Running on Time, Everlasting High Rise, Bright Expectations, The City Lights and Techno Torture. Even though I'm a Transport Fever addict now...
